src/Integration/LocalServices/SolutionBindingInformationProvider.cs

```diff
@@ -84,32 +79,18 @@ private IEnumerable<Project> GetUnboundProjects(BoundSonarQubeProject binding)
             var projectSystem = this.serviceProvider.GetService<IProjectSystemHelper>();
             projectSystem.AssertLocalServiceIsNotNull();
 
-            // Reuse the binding information passed in to avoid reading it more than once
-            return projectSystem.GetFilteredSolutionProjects().Except(this.GetBoundProjects(binding));
-        }
-
-        private IEnumerable<Project> GetBoundProjects(BoundSonarQubeProject binding)
-        {
-            if (binding == null)
-            {
-                return Enumerable.Empty<Project>();
-            }
-
-            var projectSystem = this.serviceProvider.GetService<IProjectSystemHelper>();
-            projectSystem.AssertLocalServiceIsNotNull();
-
             // Projects will be using the same solution ruleset in most of the cases,
             // projects could have multiple configurations all of which using the same rule set,
             // we want to minimize the number of disk operations since the
-            // method ca be called from the UI thread, hence this short-lived cache
+            // method can be called from the UI thread, hence this short-lived cache
             Dictionary<string, RuleSet> cache = new Dictionary<string, RuleSet>(StringComparer.OrdinalIgnoreCase);
 
             // Note: we will still may end up analyzing the same project rule set
             // but that should in marginal since it will be already loaded into memory
 
             // Reuse the binding information passed in to avoid reading it more than once
             return projectSystem.GetFilteredSolutionProjects()
-                .Where(p => this.IsFullyBoundProject(cache, binding, p));
+                .Where(p => !this.IsFullyBoundProject(cache, binding, p));
```

Comment:
Previously, the _GetUnboundProjects_ method got all of the filtered projects then called _GetBoundProjects_ to work out which ones to exclude.  Now, it just gets the filtered projects and works out which ones are not fully bound directly.